lookout/style/format/features.py

```diff
@@ -125,8 +125,7 @@ class _FeatureDiffCol(OrdinalFeature):
 
     def __call__(self, sibling: Union[VirtualNode, bblfsh.Node], node: VirtualNode
                  ) -> Iterable[int]:
-        yield min(abs(sibling.start.col - node.end.col),
-                  abs(sibling.end.col - node.start.col))
+        yield abs(sibling.start.col - node.start.col)
```

Comment:
this leaks the node length when sibling is after the node. I think considering that we should just rename diff_col and not change the computation. Maybe col_gap?